### PR TITLE
Create parser wrapper for `FR-COR->IT-SAR`

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
This PR adds a wrapper that combines the data from the IT-SACOAC and IT-SACODC exchanges.

Fixes: #4137

### CODE REVIEW

**Comment by VIKTORVAV99** [comment]
@pierresegonne This is a pretty crude solution to the problem but it works, if you have any suggestions on how to simplify things I'll be happy to implement them.

File: parsers/FR-COR_IT-SAR.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+to produce a single data list for the FR-COR->IT-SAR exchange.
+"""
+
+EXCHANGE_FUNCTION_MAP: Dict[str, Callable] = {
+    "exchange": fetch_ENTSOE_exchange,
+    "exchange_forecast": fetch_ENTSOE_exchange_forecast,
+}
+
+
+def fetch_data(
```
Comment (pierresegonne) [inline parsers/FR-COR_IT-SAR.py:24]: Clean! 

I guess the best in theory would be to modify the ENTSO-E parser to accept multiple mappings, but as it's a much bigger project, we should only do that if we another case where that happens.

File: parsers/ENTSOE.py [hunk 2]
```diff
@@ -173,9 +175,13 @@
     "BY->UA": [ENTSOE_DOMAIN_MAPPINGS["BY"], "10Y1001C--00003F"],
     "DE->DK-DK1": [ENTSOE_DOMAIN_MAPPINGS["DE-LU"], ENTSOE_DOMAIN_MAPPINGS["DK-DK1"]],
     "DE->DK-DK2": [ENTSOE_DOMAIN_MAPPINGS["DE-LU"], ENTSOE_DOMAIN_MAPPINGS["DK-DK2"]],
-    "DE->NO-NO2": [ENTSOE_DOMAIN_MAPPINGS["DE-LU"], ENTSOE_DOMAIN_MAPPINGS["NO-NO2"]],
     "DE->SE-SE4": [ENTSOE_DOMAIN_MAPPINGS["DE-LU"], ENTSOE_DOMAIN_MAPPINGS["SE-SE4"]],
-    "FR-COR->IT-CNO": ["10Y1001A1001A893", ENTSOE_DOMAIN_MAPPINGS["IT-CNO"]],
+    "DK-DK2->SE": [ENTSOE_DOMAIN_MAPPINGS["DK-DK2"], ENTSOE_DOMAIN_MAPPINGS["SE-SE4"]],
```
Comment (pierresegonne) [inline parsers/ENTSOE.py:179]: Not sure to understand why some of these changes are here?
Erroneous commit ?

**Review by pierresegonne — COMMENTED** [review]
I'll approve once I understand why some other domain mappings have been modified :)

File: parsers/ENTSOE.py (hunk 2 above)
Comment (VIKTORVAV99) [reply parsers/ENTSOE.py:179]: Ah yeah those seem to have slipped in when I merged the master branch into this, I'll clean it up.

File: parsers/FR-COR_IT-SAR.py (hunk 1 above)
Comment (VIKTORVAV99) [reply parsers/FR-COR_IT-SAR.py:24]: I don't know how you guys will handle the historical data for aggregated region but one other motivation to add it to the ENTSO-E parser would be to fix the historical exchange between Sweden and NO-NO4.

File: parsers/ENTSOE.py (hunk 2 above)
Comment (VIKTORVAV99) [reply parsers/ENTSOE.py:179]: Should be fine now.

**Comment by VIKTORVAV99** [comment]
@pierresegonne Don't really know what happened when I merged in the master branch (git is hard...)

But it looks good now.

**Review by pierresegonne — APPROVED** [review]
LGTM!

**Pull request merged**